### DISCUSSION TITLE
Totally Fucking Awesome

### DISCUSSION THREAD

**Discussion by rdegges** [asciinema/discussions#43] [Kind words]
Awesome software. I figured this needed to be said in a GitHub issue, just for the hell of it.

You rock.

**Comment by tomekwojcik** [comment]
:+1:

**Comment by fuadsaud** [comment]
hell yeah!

**Comment by decentral1se** [comment]
thoroughly agree!

**Comment by ku1ik** [comment]
If you still can reproduce this "issue" feel free to contribute: https://asciinema.org/contributing

**Comment by robgraves** [comment]
I agree, this is an amazing tool.

**Comment by dmgl** [comment]
Very useful!

**Comment by ryanpcmcquen** [comment]
Absolutely badass.

**Comment by dmgl** [comment]
No, it is not. That's your opinion. This is mine. Do not dispute - use it or not.

**Comment by marcellodesales** [comment]
(Y) Amazing!!!

**Comment by absorber** [comment]
:+1:
